src/components/instant-apps-measurement/instant-apps-measurement.tsx

```diff
@@ -33,6 +33,7 @@ export class InstantAppsMeasurement {
   @Prop() areaUnit?: __esri.AreaUnit;
   @Prop() linearUnit?: __esri.LengthUnit;
   @Prop() coordinateFormat?: string;
+  @Prop() includeCoordinates?: boolean = true;
```

Comment:
Should we have logic to update the UI when the property is programmatically set to `false` after it has been enabled? (within the configuration experience)

Reply:
@rslibed  yep missed that. Currently the coord tool is removed but the coord widget isn't created because I don't handle that situation.  I'll update.